File: 002-mockowanie/PHP/src/Util/EmailInterface.php
<?php
declare(strict_types=1);

namespace App\Util;

/**
 * Interface EmailInterface
 * @package App\Util
 */
interface EmailInterface
{
    /**
     * @param string $email
     * @return bool
     */
    public function send(string $email): bool;
}

File: 002-mockowanie/Java/src/main/java/lukaszstaniszewski/util/EmailInterface.java
package lukaszstaniszewski.util;

/**
 * The interface Email interface.
 */
public interface EmailInterface {
    /**
     * Send boolean.
     *
     * @param email the email
     * @return the boolean
     * @throws Exception the exception
     */
    public boolean send(String email) throws Exception;
}
